.pre-commit-config.yaml

```diff
@@ -3,10 +3,10 @@
 
 default_install_hook_types: [commit-msg, pre-commit]
 default_stages: [pre-commit, pre-merge-commit]
-minimum_pre_commit_version: 3.2.0
+minimum_pre_commit_version: 4.0.0
 repos:
   - repo: https://github.com/gitleaks/gitleaks.git
-    rev: v8.21.2
+    rev: v8.23.0
```

Comment:
This update breaks pre-commit locally for me because a system Go version >= 1.23 is required now and the Fedora sources come with Go 1.22.10.

Reply:
Okay, fixed it:
https://github.com/jamilraichouni/engine-room/commit/f6b4ae07adaed61f17b757c57fda93d9bc13d303

Reply:
Fedora 41 has Go 1.24 in its repos. I'd rather suggest you upgrade to that instead of just curling a tarball, because that way you get signed packages and don't have to manually bump the version number every so often :)

Reply:
Why not. Was still on Fedora 40 because v41 was not supported by Docker. This has changed and I am on Fedora 41 now. :-)